tests/python/contrib/test_ethosn/test_conv2d_transpose.py

```diff
@@ -169,6 +170,72 @@ def test_conv2d_transpose(ifm_shape, strides, kernel_size, out_channels, dtype,
     tei.verify(outputs, dtype, 1)
 
 
+@pytest.mark.skipif(
+    ethosn_api_version() == "3.2.0",
+    reason="Skip because NPU driver 22.11 does not support following cases.",
+)
+@requires_ethosn
```

Comment:
I had some trouble skipping like this in the past due to the way decorators are evaluated, might need to put the skip inside the function

Reply:
Thanks for catching it. It cause CI failure in ci_arm.